### PR TITLE
Add new privacy and terms files. Remove text from README and refer to the files instead.

### PR DESCRIPTION
**Pull request by wpschaub**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/VSTS-Bot/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- Add terms file.
- Add policy file.
- Revise README to point to these files.

@ALM-Rangers/VSTS-Bot

### CODE REVIEW

**Comment by wpschaub** [comment]
@jeffrey-opdam @Hussar77 please double check the policy and terms files to ensure you agree with the terms from a community perspective. A spell/grammar check would be recommended and ensuring that the text is 100% valid ... it's 1:30AM and I may have missed the obvious.

File: Privacy-Policy.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## DevOps VSTS Bot, Privacy Statement
+Effective date: March 10, 2018
+
+Welcome to the DevOps VSTS Bot, (hereafter “Our Plugin”).
+
+Thank you for our Plugin and making Visual Studio Team Services (VSTS) part of your daily digital conversations.
```
Comment (jeffrey-opdam) [inline Privacy-Policy.md]: Can you change it to "Thank you for using our ..."

File: Privacy-Policy.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- Our Plugin allows you to login into Visual Studfio Team Services (VSTS) and authorize Slack, Skype, and Microsoft Teams Application to enable you to access VSTS resources. We do not retain or use the data thus collected for any other purposes.
```
Comment (jeffrey-opdam) [inline Privacy-Policy.md]: Visual Studio Team Services (typo at Studfio)

File: Privacy-Policy.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Our Plugin encryts and stores your personal access token to ensure a smooth user experience when you login in future.
```
Comment (jeffrey-opdam) [inline Privacy-Policy.md]: encryts -> encrypts

**Comment by wpschaub** [comment]
@jeffrey-opdam I fixed the typos.

**Pull request merged**